src/playwrightTest.ts

```diff
@@ -222,6 +229,52 @@ export class PlaywrightTest {
     await reporterServer.wireTestListener(listener, token);
   }
 
+  async runBrowserServer(workspaceFolder: string, token: vscodeTypes.CancellationToken) {
+    if (this._browserServerWS)
+      return;
+
+    const npx = await this.findNpx();
+
+    const allArgs = [
```

Comment:
If you do `node playwright-core/cli.js ...`, that will save a process and a second.

Reply:
I originally was trying to implement it in a model-less manner, but I guess it is ok.